frontend/javascripts/oxalis/view/right-menu/trees_tab_view.js

```diff
@@ -197,9 +207,9 @@ export async function importTracingFiles(files: Array<File>, createGroupForEachF
     // Dispatch the actual actions as the very last step, so that
     // not a single store mutation happens if something above throws
     // an error
-    importActionsWithDatasetNames
-      .map(el => el.importAction)
-      .forEach(action => Store.dispatch(action));
+    _.flatten(importActionsWithDatasetNames.map(el => el.importActions)).forEach(action =>
```

Comment:
You can use `<array>.flatMap` here (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap) :)